src/utils/constants.ts

```diff
@@ -31,6 +31,12 @@ export enum PartnerAccessCodeStatusEnum {
   CODE_EXPIRED = 'CODE_EXPIRED',
 }
 
+export const hasFeatureLiveChat = (partnerAccesses) => {
```

Comment:
feels like this would go in a `utils.js` file rather than constants, but maybe one to think about for refactors as there may be others elsewhere?